app/scripts/modules/titus/src/serverGroup/configure/ServerGroupCommandBuilder.js

```diff
@@ -51,7 +51,9 @@ angular.module(TITUS_SERVERGROUP_CONFIGURE_SERVERGROUPCOMMANDBUILDER, []).factor
         targetHealthyDeployPercentage: 100,
         env: {},
         labels: {},
-        containerAttributes: {},
+        containerAttributes: {
+          'titusParameter.agent.assignIPv6Address': 'true',
```

Comment:
Should this be `false` by default?

Reply:
We default to a test environment when a new command is built, so it should be `true`. If a user switches the account to something in another environment it will flip it to false.